index.md

```diff
@@ -1,12 +1,37 @@
-# Welcome!
+# Welcome & Congratulations!
+### You just took your first step towards becoming a Virtual Intern!
 
-This MDwiki serves as an information repository for treehouses, a project of the Open Learning Exchange.  Please feel free to join us on our [Gitter chat](https://gitter.im/treehouses/Lobby).
 
-The treehouses project is divided into several parts to meet the needs of different users, including those participating in our Virtual Intern program.
+![OLE International Team](./images/OLE_International_Team.jpg)
+###### The OLE International Team. [Left to Right] Rabi, April Stone, Stefan Unterhauser, Rory Phimister, Denisia Mangue, Richard R. Rowe, Boyah J. Farah and Leonard Maximus Ireneous Mensah not pictured.
```

Comment:
Good progress @sonAndrew!
This looks a lot better.

I would like to state my opinion here and see what the other interns think; note this is not a reflection of your work.
Its to my understanding that OLE's focus is its **work and recipients**. I feel that the opening image being the crew (which can change from month to month for many of us) sets the wrong precedence.  So, it might be better to remove the picture.

I'll prompt the other interns and @dogi for their opinions on the matter.

Reply:
Yeah, I was thinking the same but felt that having a good photo of leaders looking at you would be inspiring. Would like to hear feedback as well.

Reply:
Hi @sonAndrew, nice to e-meet you! Really like the effort that went into this, some refinements that you could do just in terms of keeping the page a bit more on target would be to keep much of the font to a similar size. 

![image](https://user-images.githubusercontent.com/47016295/66678908-9c5a4080-ec75-11e9-9907-01070c53caa4.png)

Otherwise it may come off as a bit disjointed to the first people popping into our page. 

I also agree with @CalebProvost about images sometimes being better when less is more, the image of the team has allot of peeps from the main OLE team that are not part of Treehouses, awesome peeps but not part of this project :) 

The second image is also a bit overpowering because of its size, so if it needs to be there, potentially something smaller?

![image](https://user-images.githubusercontent.com/47016295/66679200-5651ac80-ec76-11e9-82a1-2f1eb0e14ad3.png)

Keep up the great work! Looking forward to welcoming you to the team!